Libplanet/Net/Swarm.cs

```diff
@@ -590,6 +590,18 @@ await DialToExistingPeers(dialTimeout, cancellationToken)
                 cancellationToken);
         }
 
+        public async Task CheckAllPeersAsync(
+            CancellationToken cancellationToken = default(CancellationToken))
+        {
+            if (cancellationToken == default(CancellationToken))
+            {
+                cancellationToken = _cancellationToken;
+            }
+
+            NetMQTransport netMQTransport = (NetMQTransport)_transport;
```

Comment:
(Trivial) I think var is enough because we cast _transport as NetMQTransport.

```suggestion
            var netMQTransport = (NetMQTransport)_transport;
```